packages/extension/src/components/pages/CreateOffer/CreateOffer.tsx

```diff
@@ -162,32 +124,34 @@ export const CreateOffer: FC = () => {
     const takerPays = urlParams.get('takerPays')
       ? parseAmount(urlParams.get('takerPays'), null, null, '')
       : null;
+    const wallet = getCurrentWallet();
 
     if (!takerGets || !takerPays) {
       setIsParamsMissing(true);
     }
 
+    if (!wallet) {
+      setIsParamsMissing(true);
+      return;
```

Comment:
Same comment

Reply:
no because the "Missing parameters" error page will be displayed because "isParamsMissing is set to true